cmd/program/program.go

```diff
@@ -100,6 +102,11 @@ func (p *Project) createFrameworkMap() {
 		packageName: echoPackage,
 		templater:   tpl.EchoTemplates{},
 	}
+
+	p.FrameworkMap["cobra"] = Framework{
```

Comment:
This is in the same prompt right now for the WEB frameworks 
Adding Cobra is entirely different - it spins up a completely new structure for an app